src/website/webview/templates/components/suggestion.html

```diff
@@ -53,14 +61,6 @@
         <button type="submit" name="create-draft-not-implemented" class="draft-color">
           C̶̖͆r̶͍̚e̵͈̐ă̷̖t̷̄͜e̶͓̓ ̴̟̏d̵̳̅r̸̮̐a̵͍͑f̵̙̋t̴̹̃
         </button>
-        {% if cached_suggestion.packages %}
```

Comment:
Doesn't this disable the scriptless workflow?

Reply:
Yeah thats right. From the commit message:

> This means that no-JS users will need to commit a status change when
removing packages, but at least that fact should be communicated well as
the page never reloads on package removal.

Reply:
Why not just remove the element when scripting is enabled?

Didn't test it but this could work:

```
<div hx-get="#" hx-swap="delete" hx-trigger="load">
    This will be removed when HTMX loads
</div>
```

Reply:
I spent some time trying to do that in htmx, but didn't figure out how. Would be cool if htmx had some init event that can be used for `hx-on`.

For now I included one line of JS code in the `base.html` file.

Reply:
I also had a look into that, but the classic seems to be the only reasonable thing.